File: test/fixture/languages/expected.80.rb
# Ruby uses '#' for line comments

#This is a comment with no space after the '#'

# This is a comment that need to have enough text to span over two lines. It
# doesn't yet; ok now it does.

# This is a two-line comment made up of two single-line comments.

    # This is a comment that's been indented
puts "Some Ruby code"

# This is a line comment right before a multi-line comment
=begin
This is a multi-line comment. Now some random text that will make the comment
large enough to fill a couple of lines at least. Btw, the '=begin' and '=end'
have to be at the start of the line with no indents.
=end
# This is a comment right after a multi-line comment

File: test/fixture/languages/expected.80.c
// '' marks the beginning or end of a section

// A single line comment with an empty selection in it. 1 2 3 4 5 6 7 8 9 0 1 2
// 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0

// A single line comment with the whole line selected. 1 2 3 4 5 6 7 8 9 0 1 2 3
// 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0

// A comment with three lines. Only the first two
// are selected

/* A multi-line comment on a single line. When this gets wrapped the next line
is indented at the same level as the '/*' of the first line */

/* This is a multi-line comment that has enough text to be wrapped but it won't be because it hasn't been selected */

// This is a line comment that has enough text to be wrapped but it won't be because it hasn't been selected.
void DoNothingWithX(int x) {
    return x;
}